ui/apps/ui/src/app/components/recommendations/recommendations.component.html

```diff
@@ -65,13 +65,13 @@ <h2 style="padding-top: 24px">Recommended resources</h2>
               [ngClass]="{
                 statistic: true,
                 'open-access':
-                  recommendation.accessRight?.toLowerCase() === 'open access',
```

Comment:
nit about this and other occurrences: how are we making sure we can safely delete the `?.` assignment?